src/Aspire.Hosting.Python/PythonProjectResourceBuilderExtensions.cs

```diff
@@ -14,21 +14,29 @@ public static class PythonProjectResourceBuilderExtensions
 {
     /// <summary>
     /// Adds a python application with a virtual environment to the application model.
-    /// The Python project should have a virtual environment set up in the project directory under the name ".venv".
-    /// <para>
-    /// The python script in the project is automatically instrumented with opentelemetry when the virtual environment
-    /// contains the opentelemetry-instrument executable. You can get this by adding the opentelemtry-distro package
-    /// to your python project. In addition to the opentelemetry-distro package, you need to add the opentelemetry-exporter-otlp
-    /// for the traces, logs, and metrics to be exported to the Aspire observability platform.
-    /// </para>
     /// </summary>
+    /// <example>
```

Comment:
nit: Can you move the `<example>` below `<remarks>`. In the description for the example could you provide a little bit of detail about the layout of the project to help people understand what "PythonProject" refers to. Like the fact that it is relative to the AppHost directory.